com.civicrm.hrjobroles/js/src/job-roles/controllers/job-roles.controller.js

```diff
@@ -511,98 +512,36 @@ define([
-                  }
-
-                  break;
-                case 'hrjc_location':
-                  if (optionGroupId === data.values[i]['option_group_id']) {
-                    // Build the contact list
-                    LocationList[data.values[i]['value']] = {
-                      id: data.values[i]['id'],
-                      title: data.values[i]['label'],
-                      value: data.values[i]['value'],
-                      is_active: data.values[i]['is_active']
-                    };
-                    // Store the Location types so we can reuse later
-                    vm.LocationsData = getActiveValues(LocationList);
-                  }
-
-                  break;
-                case 'hrjc_level_type':
-                  if (optionGroupId === data.values[i]['option_group_id']) {
-                    // Build the contact list
-                    LevelList[data.values[i]['value']] = {
-                      id: data.values[i]['id'],
-                      title: data.values[i]['label'],
-                      value: data.values[i]['value'],
-                      is_active: data.values[i]['is_active']
-                    };
-                    // Store the Level types so we can reuse later
-                    vm.LevelsData = getActiveValues(LevelList);
-                  }
-
-                  break;
-                case 'cost_centres':
-                  if (optionGroupId === data.values[i]['option_group_id']) {
-                    // Build the contact list
-                    CostCentreList.push({
-                      id: data.values[i]['id'],
-                      title: data.values[i]['label'],
-                      is_active: data.values[i]['is_active'],
-                      weight: data.values[i]['weight']
-                    });
-                    // Store the cost center list so we can reuse later
-                    vm.CostCentreList = CostCentreList;
-                  }
-
-                  break;
+              if (optionGroupId === data.values[i]['option_group_id']) {
+                if (optionGroupName === 'cost_centres') {
+                  optionsData[optionGroupName].data.push({
+                    id: data.values[i]['id'],
+                    title: data.values[i]['label'],
+                    weight: data.values[i]['weight'],
+                    is_active: data.values[i]['is_active']
+                  });
+                } else {
+                  optionsData[optionGroupName].data[data.values[i]['value']] = {
+                    id: data.values[i]['id'],
+                    title: data.values[i]['label'],
+                    value: data.values[i]['value'],
+                    is_active: data.values[i]['is_active']
+                  };
+                }
+
+                vm[optionsData[optionGroupName].storage] = getActiveValues(optionsData[optionGroupName].data);
               }
             }
           });
-
```

Comment:
That line break looks actually OK to me. It separates assignments from for-loop.

We have a Styleguide for AngularJS https://compucorp.atlassian.net/wiki/spaces/PCHR/pages/121371880/AngularJS that we try to follow.

Could you please revert it back?

Reply:
Alright noted.